dev/dll/XamlControlsResources.cpp

```diff
@@ -123,6 +129,29 @@ void SetDefaultStyleKeyWorker(winrt::IControlProtected const& controlProtected,
                     return L"ms-appx://" MUXCONTROLS_PACKAGE_NAME "/" MUXCONTROLSROOT_NAMESPACE_STR "/Themes/rs2_generic.xaml";
                 }
             }
+            else if (isInCBSPackage)
```

Comment:
CBS package will only support latest platform so you don't need to have all the platform checks.